utils/google-analytics.ts

```diff
@@ -50,32 +56,40 @@ export const useGoogleAnalytics = (): void => {
   }, []);
 };
 
-export const sendHealthCertVerifiedEvent = (data: OpenAttestationDocument): void => {
-  if (!isHealthCert(data)) {
-    return;
-  }
+export const sendSuccessfulVerificationEvent = (data: OpenAttestationDocument): void => {
+  if (utils.isRawV3Document(data)) return; // OA v3 currently not supported by verify.gov.sg
+
   try {
     ReactGA.event(EVENT_CATEGORY.VERIFIED, {
-      document_id: data.id ?? "",
-      document_type: getHealthCertType(data),
+      document_id: data.id || "",
+      document_type: getDocumentType(data),
+      issuer_identity_location: data.issuers[0].identityProof?.location || "",
+      template_name: typeof data.$template === "string" ? data.$template : data.$template?.name || "",
+      template_url: typeof data.$template === "string" ? data.$template : data.$template?.url || "",
     });
   } catch (e) {
     console.error(e);
   }
 };
 
-export const sendHealthCertErrorEvent = (data: OpenAttestationDocument, fragments: VerificationFragment[]): void => {
-  if (!isHealthCert(data)) {
-    return;
-  }
+export const sendUnsuccessfulVerificationEvent = (
+  data: OpenAttestationDocument,
+  fragments: VerificationFragment[]
+): void => {
+  if (utils.isRawV3Document(data)) return; // OA v3 currently not supported by verify.gov.sg
```

Comment:
Do we wanna log this as an error event? Or we just gonna skip this until it becomes supported, and then we will remove this line?

Reply:
Good point, I think we can log it. At the same time, I don't think we have any v3 documents generated for `verify.gov.sg` also 🤔

Reply:
We can update this when verify.gov.sg needs to be compatible with v3 documents in the future 👍